pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -125,6 +125,7 @@
         "PauliX",
         "PauliY",
         "PauliZ",
+        "Projector",
```

Comment:
Do we really support that? In general a `Projector` is defined in terms of a state vector, so we might not be able to handle it in all cases. I guess that's fine since Hermitian can also be arbitrarily large. Just stating it for awareness.